activerecord/lib/active_record/migration.rb

```diff
@@ -622,10 +622,12 @@ def announce(message)
       write "== %s %s" % [text, "=" * length]
     end
 
-    def say(message, subitem=false)
+    def say(message, subitem=false) # :nodoc:
       write "#{subitem ? "   ->" : "--"} #{message}"
     end
 
+    # Prints customized +message+ to the console along with benchmarks
+    # describing how long each step in migration took.
```

Comment:
@matthewd @rafaelfranca Please review https://github.com/prathamesh-sonpatki/rails/commit/fcddac9b9e36766eb58c144bef0cbd599a109526. Can we open this PR?